src/pages/ViewTutorInfo.js

```diff
@@ -1,17 +1,27 @@
 import "./ViewTutorInfo.css";
 import { useGetTutorByIdQuery, useUpdateTutorByIdMutation } from "../state/tutorsSlice";
+import { useGetStudentByIdQuery } from "../state/studentsSlice";
 import { useParams } from "react-router-dom"
+import { React, useEffect, useCallback, useState } from 'react'
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import { Link } from "react-router-dom"
+
+import { firebaseApp } from "../firebaseApp";
```

Comment:
you shouldn't be directly interacting with firebase, please use the slices in the state folder

Reply:
you may need to rebase as the slices have been updated since this was opened